src/app/forms/components/dropdown/dropdown.component.html

```diff
@@ -0,0 +1,25 @@
+<div *ngIf="isEditing; else view" [ngClass]="{ 'govuk-form-group': !noBottomMargin }">
+  <h1 *ngIf="label" class="govuk-label-wrapper">
+    <label class="govuk-label govuk-label--m" for="{{ name }}">{{ label }}</label>
+  </h1>
+
+  <div *ngIf="hint" id="{{ name }}-hint" class="govuk-hint">{{ hint }}</div>
+
+  <app-field-error-message [error]="error" [name]="name"></app-field-error-message>
+
+  <select
```

Comment:
There should be a `select` component already in the code base

Reply:
What you're referring to is the `app-autocomplete` and although it has a `<select>` element inside, it is an auto-complete field, different to a dropdown.

Reply:
I thought about extending that to add a toggle to change functionality between an autocomplete and a dropdown, but that component contains a lot of boilerplate code for the `enhanceSelectElement` which makes it difficult to extend.

Reply:
Actually Taha, there is a select component. The App-Autocomplete is a different component. Literally in the same folder as this there should be a component called Select

Reply:
Oh my... Guess my coffee wasn't harsh enough today. Just saw it and will use that instead.